ui/components/app/transaction-detail-item/transaction-detail-item.component.js

```diff
@@ -15,26 +16,32 @@ export default function TransactionDetailItem({
   detailTotal = '',
   subTitle = '',
   subText = '',
+  boldHeadings = true,
+  flexWidthValues = false,
 }) {
   return (
     <div className="transaction-detail-item">
       <div className="transaction-detail-item__row">
         <Typography
           color={detailTitleColor}
-          fontWeight={FONT_WEIGHT.BOLD}
+          fontWeight={boldHeadings ? FONT_WEIGHT.BOLD : FONT_WEIGHT.NORMAL}
```

Comment:
ideally designs should have this alined across screens

Reply:
I agree. I had to customize it in this case because we are adding custom information for this one network. This can be modified when we rework the confirm screen redesigns in Jan/Feb